### ISSUE TITLE
Loss curve looks good,but performance bad

### ISSUE THREAD

**Issue by mitumao-DD** [open-mmlab/mmdetection#1538]
I  want to finetune RetianNet model on my own dataset.  I changed the num_cls and dataset config in cfg file and Classes tuple in COCO.py 

My dataset is a subset of COCO!

The loss curve looks good ,but when I test it on test dataset, its mAP is much lower than the original model.

[losses.pdf](https://github.com/open-mmlab/mmdetection/files/3721460/losses.pdf)

Then , I guess if my dataset is not big enough and my model can still be trained. In order to check this idea, I use the original model (change the num_cls back to 81) and set the learning rate to 2e-8  to see the value of training loss . However, the loss of original model in the first batch is much higher than my finetune model. 

So,I am very confused about this problem. My finetune model get lower losses than orginal model, but its performance is worser than original one.  

Could you give me some advice? Thanks a lot!

**Comment by flyfj** [comment]
@mitumao-DD can you provide some insights on this? came across a similar issue
